### PR TITLE
issue/188-fix-quickstart-pipeline

### PR DESCRIPTION
**Pull request by Hugo-Inmanta**
Closes https://github.com/inmanta/infra-tickets/issues/188

### CODE REVIEW

File: Networking/SR Linux/ci/Jenkinsfile [hunk 1]
```diff
@@ -84,7 +83,9 @@ pipeline {
                             # Ensure we have the latest version of the srlinux image
                             sudo docker pull ghcr.io/nokia/srlinux:latest
                             sudo clab deploy -t containerlab/topology.yml --reconfigure
-                            ci/do_test_deployment_and_verify.sh ${version}
+                            # Get python version used to build the container
+                            python_version=$(docker inspect clab-srlinux-inmanta-server  | grep PYTHON_VERSION |  cut -d"=" -f2- | cut -d'"' -f1)
```
Comment (sanderr) [inline Networking/SR Linux/ci/Jenkinsfile:87]: Cool! This is better than what I had in mind! Do you know where the env var comes from though? Can we rely on it?

**Review by sanderr — APPROVED** [review]

File: Networking/SR Linux/ci/Jenkinsfile (hunk 1 above)
Comment (Hugo-Inmanta) [reply Networking/SR Linux/ci/Jenkinsfile:87]: I think it is set here https://github.com/inmanta/irt/blob/7bebdec6640558cd66630fa379c1cf47ba3a13ef/irt/release/build_docker_image.py#L280

File: lsm-srlinux/ci/Jenkinsfile [hunk 2]
```diff
@@ -149,8 +149,12 @@ pipeline {
                             # Ensure we have the latest version of the srlinux image
                             sudo docker pull ghcr.io/nokia/srlinux:latest
 
+                            sudo clab deploy -t containerlab/topology.yml --reconfigure
+                            # Get python version used to build the container
+                            python_version=$(docker inspect clab-srlinux-inmanta-server  | grep PYTHON_VERSION |  cut -d"=" -f2- | cut -d'"' -f1)
```
Comment (arnaudsjs) [inline lsm-srlinux/ci/Jenkinsfile:154]: I like it, but I am wondering whether it works correctly. I tried this command on an iso7 docker images that was present on my system and I get two times the version number:

```
$ python_version=$(docker inspect 4dc3990f106c | grep PYTHON_VERSION | cut -d"=" -f2- | cut -d'"' -f1)
$ echo ${python_version}
3.9
3.9
```

Did you test this?

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

**Comment by Hugo-Inmanta** [comment]
Tested against
- [network-quickstart - latest](https://jenkins.inmanta.com/job/integration-tests/job/network-quickstart/job/issue%252F188-jenkinsfile-test-branch/12/)
- [network-quickstart - 2023.1](https://jenkins.inmanta.com/job/integration-tests/job/network-quickstart/job/issue%252F188-jenkinsfile-test-branch/15/parameters/)
- [quickstart-lsm - dev7](https://jenkins.inmanta.com/job/integration-tests/job/quickstart-lsm/job/issue%252F188-jenkinsfile-test-branch/7/parameters/)

**Review by sanderr — APPROVED** [review]

**Review by arnaudsjs — APPROVED** [review]

**Pull request merged**